crypto/perlasm/README

```diff
@@ -35,7 +35,7 @@ Functions of interest are:
 &function_begin_B(name,extra)	Same as normal function_begin but no pushing.
 &function_end(name)		Call at end of function.
 &function_end_A(name)		Standard pop and ret, for use inside functions
-&function_end_B(name)		Call at end but with poping or 'ret'.
+&function_end_B(name)		Call at end but with pop'ing or 'ret'.
```

Comment:
> I didn't choose "popping" on purpose, to make the "pop" statement more apparent.

Revisiting this line, it does not seem like a real improvement to me. I will just change it to

```
Call at end but with 'pop' or 'ret'.
```